docs-src/python/backgroundcheck-boilerplate-self-hosted-worker.md

```diff
@@ -0,0 +1,49 @@
+---
+id: backgroundcheck-boilerplate-self-hosted-worker
+title: Customize Client options
+sidebar_label: Self-hosted Client options
+description: Configure the Temporal Client with the specific IP Address of the Temporal Server on your network.
+tags:
+- worker
+- self-hosted
+- developer guide
+---
+
+<!-- DO NOT EDIT THIS FILE DIRECTLY.
+THIS FILE IS GENERATED from https://github.com/temporalio/documentation-samples-python/blob/bgc/backgroundcheck_boilerplate/self_hosted_worker/main_dacx.py. -->
+
+Set IP address, port, and Namespace in the Temporal Client options.
+
+:::copycode Sample application code
+
+The following code sample comes from a working and tested sample application.
+The code sample might be abridged within the guide to highlight key aspects.
+Visit the source repository to [view the source code](https://github.com/temporalio/documentation-samples-python/blob/bgc/backgroundcheck_boilerplate/self_hosted_worker/main_dacx.py) in the context of the rest of the application code.
+
+:::
+
+```python
+import asyncio
+import os
+
+from temporalio.client import Client, TLSConfig
+from temporalio.worker import Worker
+
+from activities.ssntraceactivity_dacx import ssn_trace_activity
+from workflows.backgroundcheck_dacx import BackgroundCheck
+
+
+
```

Comment:
Inconsistent on whether you have `# ...` between imports and `main` vs some number of blank lines